cluster/node-pools/master-ubuntu-default/userdata.yaml

```diff
@@ -98,7 +98,7 @@ write_files:
         hostNetwork: true
         containers:
         - name: kube-apiserver
-          image: registry.opensource.zalan.do/teapot/kube-apiserver:v1.13.7
+          image: nonexistent.zalan.do/teapot/kube-apiserver:fixed
```

Comment:
Would it make sense to use something like `local-cache` instead of `nonexistent` as a more descriptive name?

Reply:
I think `nonexistent` is more descriptive because the reason for it is tagged with a registry which doesn't exist so that it cannot be pulled.